src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const temp = Object.entries(robot);
+  const obj = Object.create({});
```

Comment:
```suggestion
  const obj = {};
```
используй короткий формат записи